tests/requirements-gevent-starlette.txt

```diff
@@ -4,6 +4,6 @@ gevent>=1.4.0
 mock>=2.0.0
 pyramid>=2.0.1
 pytest>=4.6
-starlette>=0.12.13
+starlette>=0.38.2
```

Comment:
This change is unnecessary if the CI pipeline that uses this requirements file works.

Reply:
Done.